kodilla-good-patterns/src/main/java/com/kodilla/good/patterns/challenges/challenge5/flight/FlightSearchProcessor.java

```diff
@@ -0,0 +1,39 @@
+package com.kodilla.good.patterns.challenges.challenge5.flight;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class FlightSearchProcessor {
+    private static FlightRepository flightRepository;
```

Comment:
jesli deklarujesz te zmienna w konstruktorze, to powinna byc 
private lub
private final